### PR TITLE
Curriculum launch 2024 - Update Computer Vision landing page

### PR DESCRIPTION
**Pull request by kelbyhawn**
Updates the https://code.org/curriculum/computer-vision page for the 2024 curriculum launch. Uses the `curriculum-launch-2024` DCDO flag to show the updated version of the page.

**Note:** when reviewing turn off the whitespace changes to hide the old version of the page where nothing changed.

## Links
Jira ticket: [ACQ-1961](https://codedotorg.atlassian.net/browse/ACQ-1961)
Figma mock: [here](https://www.figma.com/design/HwXIyBGQ0b2ZI213sWb3cK/2024-Curriculum-Launch?node-id=2424-20914&t=WYtTg918El6KfrDt-1)

## Testing story
Tested locally that the DCDO logic works

## Followup
I'll clean up this page's old version and logic after the launch in this ticket: [ACQ-2006](https://codedotorg.atlassian.net/browse/ACQ-2006)

----

## DCDO is `true` (updated version of the page)
True

## DCDO is `false` (current version of the page)
False

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 1]
```diff
@@ -10976,22 +10976,22 @@
       heading: "Why teach computer vision?"
       desc: "Empower students to solve real-world challenges, foster innovation, and explore AI's ethical implications in visual data."
       real_world:
-        heading: "Real-world applications"
+        title: "Real-world applications"
```
Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml:10979]: I updated these string names to match what I did for a similar sections in the `game_design_page` namespace.

File: pegasus/sites.v3/code.org/public/curriculum/computer-vision.haml [hunk 2]
```diff
@@ -1,75 +1,137 @@
...
+        = view :"computer_vision/computer_vision_teach"
+
+  %section.bg-neutral-light
+    .wrapper.centered
+      %h2=hoc_s("computer_vision_page.learn_more.heading")
+      = view :"computer_vision/computer_vision_learn_more"
+      %hr.light
+      %h3=hoc_s("computer_vision_page.learn_more.discover.heading")
+      %p=hoc_s("computer_vision_page.learn_more.discover.desc")
+      %a.link-button.secondary{href: "/ai"}
```
Comment (TurnerRiley) [inline pegasus/sites.v3/code.org/public/curriculum/computer-vision.haml:63]: Nit: Should this have `.white` (or some other color) to contrast the gray background?

**Review by TurnerRiley — APPROVED** [review]

File: pegasus/sites.v3/code.org/public/curriculum/computer-vision.haml (hunk 2 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/curriculum/computer-vision.haml:63]: Nope! The secondary buttons have a transparent bg, but there is a `.white` version that turns the outline and text white if it's on a dark background.

Comment (TurnerRiley) [reply pegasus/sites.v3/code.org/public/curriculum/computer-vision.haml:63]: Oh gotcha!

**Pull request merged**